.github/workflows/msbuild.yml

```diff
@@ -15,26 +15,27 @@ jobs:
         ## Macos jobs tend to stall -- not including them for now
         ## Windows jobs fail on some lit tests and sometimes fail even to
         ## setup a work environment (e.g. dotnet installation)
-        os: [ubuntu-latest]
+        ##os: [ubuntu-latest]
         ##os: [ubuntu-latest, windows-latest, ubuntu-16.04]
         ##os: [ubuntu-latest, macos-latest]
-        ##os: [ubuntu-latest, macos-latest, windows-latest, ubuntu-16.04]
+        os: [
+             ubuntu-latest,
+             macos-latest,
+           ##  windows-latest,
```

Comment:
So, Windows still doesn't work? Does this mean the `runner.os == 'Windows'` test below will always be `false` for now?

Reply:
For the moment, yes. Still needs some debugging to build a release on Windows.